ts/miso/util.ts

```diff
@@ -12,6 +12,14 @@ export function callFocus(id: string, delay: number): void {
   delay > 0 ? setTimeout(setFocus, delay) : setFocus();
 }
 
+export function callSelect(id: string, delay: number): void {
+  var setSelect = function () {
+    var e : HTMLInputElement = document.querySelector('#' + id);
+    if (e && e.select) e.select();
```

Comment:
As @jhrcek mentions, `getElementById` will be a faster lookup. The TypeScript type hierarchy is very rigid for DOM elements. We should be able to `as`-cast into `HTMLInputElement` and use a dynamic property check to avoid issues. Below should act as a safe runtime check, but also satisfy the typechecker.

```typescript
if (e && typeof e['select'] === 'function') (e as HTMLInputElement).select();
```

Also note the `#` prefix will need to be dropped above when using `getElementById`.